src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import './Todo.css';
+const TodoForm = props => {
+  return (
+    <React.Fragment>
+      <form onSubmit={props.addItem}>
+        <input
+          type="text"
+          name="inputText"
+          placeholder="Add New Todo"
+          value={props.inputText}
+          onChange={props.handleChange}
+        />
+        <button type="submit">Add Todo</button>
+        <button>Clear Completed</button>
+      </form>
+    </React.Fragment>
```

Comment:
Why use a fragment here? Your form is the parent element and has no siblings; the fragment adds nothing dom/styling-wise, so it's unnecessary here.